### PR TITLE
docs: add paypal hook endpoint

### PR DESCRIPTION
**Pull request by ghiottimartin**
- Added paypal webhook endpoint to the documentation

### CODE REVIEW

**Comment by ghiottimartin** [comment]
This fix this issue: https://github.com/medusajs/medusa/issues/4954

File: www/docs/content/plugins/payment/paypal.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ const plugins = [
 ]
 ```
 
+The endpoint for PayPal webhook integration with your Medusa backend should be set to `{BACKEND_URL}/paypal/hooks`. Make sure to replace `{BACKEND_URL}` with the URL to your backend.
```
Comment (shahednasser) [inline www/docs/content/plugins/payment/paypal.md]: Should this be set in paypal's configuration? If so, I would add this under the Prerequisites section

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution! Just a comment on the text's placement

File: www/docs/content/plugins/payment/paypal.md (hunk 1 above)
Comment (ghiottimartin) [reply www/docs/content/plugins/payment/paypal.md]: Yes, no problem!

**Pull request merged**